src/modules/position_estimator_inav/module.mk

```diff
@@ -32,17 +32,25 @@
 ############################################################################
 
 #
-# Makefile to build position_estimator_inav
+# @brief Makefile to build position_estimator_inav
+#
+# @author Anton Babushkin <rk3dov@gmail.com>
+# @author Nuno Marques <n.marques21@hotmail.com>
 #
-
 MODULE_COMMAND	 	= position_estimator_inav
-SRCS		 	= position_estimator_inav_main.c \
+SRCS		 	= position_estimator_inav_main.cpp \
 			position_estimator_inav_params.c \
-			inertial_filter.c
+			inertial_filter.cpp
 
 MODULE_STACKSIZE = 1200
 
-ifeq ($(PX4_TARGEGT_OS),nuttx)
-EXTRACFLAGS = -Wframe-larger-than=3800
+EXTRACFLAGS = -Wno-float-equal
+
+ifneq ($(USE_GCC), 0)
```

Comment:
https://github.com/PX4/Firmware/blob/2afa499c662aaaf9c1108f2c1790fe91ca3b03e3/src/systemcmds/tests/module.mk#L45-L50. They were also required here as it was issuing some warnings/errors.